run/send_matrix.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+import sys
+import time
+from matrix_client.errors import MatrixHttpLibError
+from matrix_client.client import MatrixClient
+from matrix_client.api import MatrixHttpApi
+
+if len(sys.argv) != 6:
+    sys.stderr.write('usage: %s HOMESERVER_URL USERNAME PASSWORD ROOM_ID MESSAGE_TEXT\n' % sys.argv[0])
+    sys.exit(-1)
+
+(homeserver, username, password, room_id, message) = sys.argv[1:6]
+
+if homeserver.endswith('/'):
+    homeserver = homeserver[:-1]
```

Comment:
If there should be no trailing /, the examples in the documentation and sample config file probably shouldn't have one.

Reply:
Agreed that matrixcli appears to be a graveyard project, and that send_matrix.py is a much smoother solution. Bravo.